Other proposals (#3)
Malware-Research/Resources: README.md

 - Want to know more about Process Injection, anti-analysis techniques, packers, and how to unpack them? Check out [OALabs’ YouTube channel](https://www.youtube.com/channel/UC--DwaiMV-jtO-6EvmKOnqg).
  
 - Want to know more about file formats or hash collisions? Check [Corkami’s GitHub repositories](https://github.com/corkami).
 
 - If you skills are enough advanced and you're really curious about APT groups then you'll have fun analyzing these samples used by most famous APTs groups: [Malware Collection] (iec56w4ibovnb4wc.onion.si)  . Please be careful, these samples are armed!!
 

>  - If you want to share your samples or you need some IoCs for your research then take a look here: [CyberSaiyan Sharing Platform] (https://github.com/CyberSaiyanIT/InfoSharing/blob/master/README.md#componente-consumer) It's in Italian but english version is coming soon 

Author: You're right. Better to remove a non-english resource.

>  - How can you complete your research, recognise patterns and samples and extract IoCs if you don't use Yara rules? Then better to use this great repo built by Florian Roth: [Neo23x0 Yara Repo](https://github.com/Neo23x0/signature-base/tree/master/yara) 

Reviewer: ```suggestion
 - Looking for a set of Yara rules to enrich your detection? Check out this collection of Yara rules by Florian Roth: [Neo23x0 Yara Repo](https://github.com/Neo23x0/signature-base/tree/master/yara) 
```

Author: Looks better :-)

 - Want to know more about Process Injection, anti-analysis techniques, packers, and how to unpack them? Check out [OALabs’ YouTube channel](https://www.youtube.com/channel/UC--DwaiMV-jtO-6EvmKOnqg).
  
 - Want to know more about file formats or hash collisions? Check [Corkami’s GitHub repositories](https://github.com/corkami).

>  
>  - If you skills are enough advanced and you're really curious about APT groups then you'll have fun analyzing these samples used by most famous APTs groups: [Malware Collection] (iec56w4ibovnb4wc.onion)  . Please be careful, these samples are armed!!

Reviewer: ```suggestion
```

>  - If you skills are enough advanced and you're really curious about APT groups then you'll have fun analyzing these samples used by most famous APTs groups: [Malware Collection] (iec56w4ibovnb4wc.onion.si)  . Please be careful, these samples are armed!!

Reviewer: This link is unavailable, We can give a pass to this suggestion :)

Author: oh...gosh! then let's replace it with this: iec56w4ibovnb4wc.onion

>  - What do you need to be a good researcher on APT groups? Surely a list of most famous Apt groups and all details related them: [APT Groups and Operations Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vTheajUWzRhTK0XhSI3_RnYVtUJvl8mlX8HlThPyCJGK1g5SBecgS78O1oeTFQxDYS0oWlKTg2pNLyb/pubhtml#)

Reviewer: ```suggestion
 - A list of popular APT groups and details about them: [APT Groups and Operations Spreadsheet](https://docs.google.com/spreadsheets/d/e/2PACX-1vTheajUWzRhTK0XhSI3_RnYVtUJvl8mlX8HlThPyCJGK1g5SBecgS78O1oeTFQxDYS0oWlKTg2pNLyb/pubhtml#)
```

 
 - Looking for a set of Yara rules to enrich your detection? Check out this collection of Yara rules by Florian Roth: [Neo23x0 Yara Repo](https://github.com/Neo23x0/signature-base/tree/master/yara) 
 

>  - What about Cyber Threat Intelligence? Don't you need a platform to collect any useful information about Apt groups, IoCs and info? Then this is waht you need: [OpenCti](https://opencti.io)

Reviewer: ```suggestion
 - A platform to collect useful information about APT groups, IOCs, and more: [OpenCti](https://opencti.io)
```
